### DISCUSSION TITLE
Linux Skia open app maximized and undecorated

### DISCUSSION THREAD

**Discussion by tompinnacle** [unoplatform/uno#14860] [Q&A]
The title pretty much says it all.

I have an app working using Uno and C# running on a linux (Raspberry Pi) that I can maximize and undecorate manually once running, but I need it to start like that by default.

It's Skia GTK in case it makes a difference

**Comment by jeromelaban** [comment]
Thanks for the question! 

I'm assuming that you use the `ApplicationView.GetForCurrentView().TryEnterFullScreenMode()` API to get there, or are you using a different API?
